src/components/offers/details/OffersDetails.js

```diff
@@ -35,7 +37,7 @@ export const OfferDetails = (props) => {
                                 </Box>
                                 <Box m={1}>
                                     <Typography variant="h6" color="textSecondary">
-                                        {offer.company}
+                                        {offer.organization?.name}
```

Comment:
JS też ma takie null checki?

Reply:
Gdzieś u Kuby to widziałem i tak działa, więc na to wychodzi, że ma. Dziwne, nie? Choć to pewnie bardziej undefined checki XD